### DISCUSSION TITLE
Possible to deploy wave app on a Kubernetes cluster?

### DISCUSSION THREAD

**Discussion by Alex-Wenner-FHR** [h2oai/wave#1055] [Q&A] [answered]
I was wondering if it would be possible to deploy a wave app as a docker image onto a K8 cluster? If so, would this be a complicated process?

I would imagine you could get to a point where you have a number of Wave apps running on different nodes, within the same cluster, that pull different images. This would be very cool!

**Comment by mturoci** [comment, marked as answer]
Sure, we actually offer our own Appstore solution exactly for this need. Check out https://www.h2o.ai/ and make sure you try our [free 14-day trial](https://www.h2o.ai/freetrial/#form).

**Reply by Alex-Wenner-FHR** [reply]
Ah I see, very neat! I will explore this a bit. Thanks!
